py_zipkin/logging_helper.py

```diff
@@ -30,9 +30,10 @@ def emit(self, record):
 class ZipkinLoggingContext(object):
     """A logging context specific to a Zipkin trace. If the trace is sampled,
     the logging context sends serialized Zipkin spans to a transport_handler.
-    This is meant to only be used server-side, so the logging context sends
-    a single "server" span, as well as all local child spans collected within
-    the server context.
+
+    The trace can represent
+        1) server and contains root server span and child client spans (optionally)
+        2) client and contains root client span only
```

Comment:
I think this new comment is not quite correct. The original comment was saying basically that the root span is a `server` span, but the local child spans can be client and/or server. In fact, our "local" spans use both cs/cr and ss/sr annotations.

You can still have this with a client root span. Imagine a application that will download some file, and also has some local method for validation that we instrumented.  We might have

```
cs:          download_file
cs/sr/ss/sr: validate_parameters_locally
cs:          fetch_file_from_server
cr:          fetch_file_from_server
cr:          download_file
```

You could even imagine child server spans in there if somewhere in the middle we check some queue for new data.

All of this is pretty unfamiliar, so i'm open to any other opinions on this matter.